src/content/changes/highlights/2022-01-14.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2022-01-14T10:00:00
+title: Highlights for the week ending January 14, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [falco-app](https://github.com/giantswarm/falco-app) [v0.2.0](https://github.com/giantswarm/falco-app/blob/master/CHANGELOG.md#020---2021-12-17) Update to upstream charts: Falco 1.16.2/0.30.0, exporter 0.6.3/0.6.0, sidekick 0.4.4/2.24.0.
+- Dex monitoring is now available through our Public Grafana ( [Docs here](https://docs.giantswarm.io/advanced/configure-dex-in-your-cluster/) )
+
+## Documentation
+
+- Better clarification about Security offering compared to other cloud native packs
+- Use apiextensions-application v0.3.0 in CRD docs 
```

Comment:
The documentation changes are not worth a highlight IMO. I would remove the entire block.